operator/pkg/utils/patch/patch.go

```diff
@@ -70,7 +70,7 @@ func parseArgsFor(podSpec *v1.PodSpec) map[string]string {
 	result := map[string]string{}
 	for _, arg := range podSpec.Containers[0].Args {
 		kv := strings.Split(arg, "=")
-		result[kv[0]] = result[kv[1]]
+		result[kv[0]] = kv[1]
```

Comment:
Nice! So earlier to this fix, changes to the flags from spec weren't working I assume ?

Reply:
yes, this broke the merge functionality.